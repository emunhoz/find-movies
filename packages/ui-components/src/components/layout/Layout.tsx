import React from 'react'
import * as S from './Layout.style'

interface ILayout {
  children: React.ReactNode
}

export const Layout = ({ children }: ILayout) => <S.Layout>{children}</S.Layout>
