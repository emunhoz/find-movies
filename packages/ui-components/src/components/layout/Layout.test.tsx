import React from 'react'
import { render } from '../../utils/test-utils'
import { Layout } from './Layout'

describe('[Component]: Layout', () => {
  test('render the component', () => {
    const { asFragment } = render(<Layout><h1>Ol√°</h1></Layout>)
    expect(asFragment()).toMatchSnapshot()
  })
})
