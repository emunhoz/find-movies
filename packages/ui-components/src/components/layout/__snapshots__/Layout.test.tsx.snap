// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component]: Layout render the component 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ cgOHzx"
  >
    <h1>
      Ol√°
    </h1>
  </div>
</DocumentFragment>
`;
